src/components/Switchers/index.examples.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react'
+import {storiesOf} from '@kadira/storybook'
+import StaticSwitcher from '.'
+
+const decoratorClasses = 'flex flex-column content-center justify-around flex-wrap items-center vh-100 bg-navy'
```

Comment:
I'm planning to refactor all these decorator classes in the future to avoid the repetition, but this is fine for now.
https://github.com/eggheadio/egghead-instructor-center/issues/119